src/scan_shadows_filter.cpp

```diff
@@ -0,0 +1,169 @@
+/*
+/* Software License Agreement (BSD License)
+ *
+ * Copyright (c) 2008-2021, laser_filters authors
```

Comment:
Same comments - leave radu in the license line

Reply:
Done